maven-extension/src/main/java/io/opentelemetry/maven/OpenTelemetrySdkService.java

```diff
@@ -184,6 +184,6 @@ protected Resource getMavenResource() {
 
   /** Returns the {@link ContextPropagators} for this {@link OpenTelemetry}. */
   public ContextPropagators getPropagators() {
-    return openTelemetrySdk.getPropagators();
+    return GlobalOpenTelemetry.getPropagators();
```

Comment:
why is this better than using a configured instance? What is setting the global?

Reply:
It seems like relying on the global instance, and the initialization thereof would be a bad idea in almost every case.

Reply:
As the same instance is set globally, it seemed preferable to use that approach.

The problem I found occurs when `openTelemetrySdk` is null because the endpoint was not specified on the command line.

Happy to resolve with an alternative approach, such as a null check and returning the noop propagators

Reply:
you could pre-initialize it to the no-op implementation : `OpenTelemetry.noop()`

Reply:
ah, will try that. thanks

Reply:
I've updated it with that approach